integrations/pytorch/notebooks/classification.ipynb

```diff
@@ -34,9 +34,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
nit: all outputs must be cleared from notebook cells before committing (`cell output -> clear all` or something)

Reply:
great spot!